### PR TITLE
Update dbcli related software. Add python37 support.

### PR DESCRIPTION
**Pull request by xeron**
#### Description

* mycli: 1.17.0
* pgcli: 1.10.3
* py-cli-helpers: 1.0.2
* py-pgspecial: 1.11.2
* py-prompt_toolkit: 1.0.15
* py-pymysql: 0.9.2 (closes https://trac.macports.org/ticket/56902)
* py-sqlparse: 0.2.4

###### Type(s)

- [x] update
- [x] enhancement

###### Tested on

macOS 10.13
Xcode 9.4

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: databases/mycli/Portfile [hunk 1]
```diff
@@ -16,20 +16,23 @@ long_description    ${description}
 
 homepage            http://mycli.net
 
-checksums           rmd160  baed20c0ef502c73b66e436c3f5d7604666949c3 \
-                    sha256  76e1ba3bca1bcda5c958c89a5e35e587f69fefab7ada00fbccdcdd5d7c5424af
+checksums           rmd160  60e893719fe8a4182bd05426966637889cf29fa2 \
+                    sha256  ed4ca1b595a5b0d319f9743277c34039d7ab5440d042e8bfa22304b89ee0d5f9
```
Comment (pmetzger) [inline databases/mycli/Portfile]: It is now customary to put in a "size" field as well.

File: databases/pgcli/Portfile [hunk 2]
```diff
@@ -16,20 +16,23 @@ long_description    ${description}
 
 homepage            http://pgcli.com
 
-checksums           rmd160  922f2cf729ab173edef221f39005d9e2d4d1c81c \
-                    sha256  e42291b45e762176218e2e937e7d1603f6f8eb32d17fd97907bb625d003e93dd
+checksums           rmd160  61fbaba17dd699f0fa0faf1a042a4525c7dead23 \
+                    sha256  75a5012a43462bf12d48baeb6bf25378dc117f9380af837112c3cf95fd945ee3
```
Comment (pmetzger) [inline databases/pgcli/Portfile]: It is now customary to put in a "size" field in addition to the rmd160 and sha256.

File: python/py-cli-helpers/Portfile [hunk 3]
```diff
@@ -14,14 +14,14 @@ long_description    CLI Helpers is a Python package that makes it easy \
...
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            cli_helpers-${version}
 
-checksums           rmd160  d9a431a99f9a605bab1837d385c5cf7d41549d0a \
-                    sha256  55353117960700dfbe000a71cda0bad1ac865e3a9999f1fa81047fa9e1322d42
+checksums           rmd160  4e1f6bb54c161a0a4eb7dc9e27eb574470e2749c \
+                    sha256  f77837c5fbcbea39e0cb782506515459a0da75465489bae35e46da7f51c5b9fc
```
Comment (pmetzger) [inline python/py-cli-helpers/Portfile]: It is now customary to put in a "size" field in addition to the rmd160 and sha256.

File: python/py-pgspecial/Portfile [hunk 4]
```diff
@@ -13,15 +13,14 @@ description         Meta-commands handler for Postgres Database
...
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           md5     6e617cb1915f414d6ecdc075d77d5f87 \
-                    rmd160  ba666f257b7f54c1bb2be05b91e9c41f10fc1c5b \
-                    sha256  eadb0108cdbcf8b38a69bcc9e403b352dbd6d30622e417f48e659180150ee1b6
+checksums           rmd160  35f6e53c5faa0a3d8da4d18ef5e96216f3ebf258 \
+                    sha256  60e81ed02c3a2d43a4eb70d5bbdeea7b0c5244580d3017edf89a6fd66e6503fb
```
Comment (pmetzger) [inline python/py-pgspecial/Portfile]: It is now customary to put in a "size" field in addition to the rmd160 and sha256.

File: python/py-prompt_toolkit/Portfile [hunk 5]
```diff
@@ -4,22 +4,22 @@ PortSystem          1.0
...
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  c0c4aa0c917981767d45abf76fdff00c0fb9aa8d \
-                    sha256  cc66413b1b4b17021675d9f2d15d57e640b06ddfd99bb724c73484126d22622f
+checksums           rmd160  195e01e80e62e631f7c4b12d4f2cd5d4481a0752 \
+                    sha256  858588f1983ca497f1cf4ffde01d978a3ea02b01c8a26a8bbc5cd2e66d816917
```
Comment (pmetzger) [inline python/py-prompt_toolkit/Portfile]: It is now customary to put in a "size" field in addition to the rmd160 and sha256.

File: python/py-pymysql/Portfile [hunk 6]
```diff
@@ -4,22 +4,22 @@ PortSystem          1.0
...
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            PyMySQL-${version}
 
-checksums           rmd160  6bb48cc4b0028666b94bfedb36f3841bb9f34a10 \
-                    sha256  32da4a66397077d42908e449688f2ec71c2b18892a6cd04f03ab2aa828a70f40
+checksums           rmd160  6c5c1015e61591e59927b4b8588acf274b6e6edd \
+                    sha256  9ec760cbb251c158c19d6c88c17ca00a8632bac713890e465b2be01fdc30713f
```
Comment (pmetzger) [inline python/py-pymysql/Portfile]: It is now customary to put in a "size" field in addition to the rmd160 and sha256.

File: python/py-sqlparse/Portfile [hunk 7]
```diff
@@ -13,16 +13,16 @@ description         Non-validating SQL parser
...
 categories-append   textproc
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  6264e56cbfdca0273b11c0f6a3334891ac9c574b \
-                    sha256  d446296b2c26f9466860dd85fa32480bec523ab96bda8879262c38e8e8fbba21
+checksums           rmd160  cbc1000e0fea6334d1981e6b5bee1d9dd95b8102 \
+                    sha256  ce028444cfab83be538752a2ffdb56bc417b7784ff35bb9a3062413717807dec
```
Comment (pmetzger) [inline python/py-sqlparse/Portfile]: It is now customary to put in a "size" field in addition to the rmd160 and sha256.

**Comment by xeron** [comment]
Updated.

**Comment by pmetzger** [comment]
Thank you, @xeron!

**Pull request merged**